include/dlaf/matrix/panel.h

```diff
@@ -305,8 +304,12 @@ struct Panel<axis, const T, D, StoreTransposed::No> {
 protected:
   using ReadWriteSenderType = typename BaseT::ReadWriteSenderType;
 
+  bool isBoundToMatrixWithSubTileOffset() const noexcept {
```

Comment:
`WithSubTileOffset`: why not just with `WithNonZeroOffset` or, abbreviated, `WithOffset`?

Reply:
Do you mean going for `isBoundToMatrixWithOffset` or just `WithSubTileOffset`?

One reason I would go for being more verbose (former one), is because I want to keep it clearly separated from the panel functionality where you can have sub-tile offset (i.e. `setRangeStart`) while this refers to the fact that the panel might be bound to a matrix with a sub-tile offset (and `setRange` would not work for the panel). Not sure it helps in clarifying...

Reply:
I meant the former, i.e. `isBoundToMatrixWithOffset`. In other words, my question was: why is `SubTile` in the name? What information does it convey?